modules/gapi/CMakeLists.txt

```diff
@@ -328,18 +328,11 @@ if(HAVE_GAPI_ONEVPL)
     ocv_target_include_directories(${the_module} SYSTEM PRIVATE ${OPENCL_INCLUDE_DIRS})
   endif()
 
-  if(UNIX)
-    if(WITH_VA)
-      include("${OpenCV_SOURCE_DIR}/cmake/OpenCVFindVA.cmake")
-      if(VA_INCLUDE_DIR)
-        ocv_target_include_directories(${the_module} SYSTEM PRIVATE ${VA_INCLUDE_DIR})
-        ocv_target_include_directories(opencv_test_gapi SYSTEM PRIVATE ${VA_INCLUDE_DIR})
-        ocv_target_link_libraries(${the_module} PRIVATE ${VA_LIBRARIES})
-        ocv_target_link_libraries(opencv_test_gapi PRIVATE ${VA_LIBRARIES})
-      endif()
-    else(WITH_VA)
-      message(FATAL_ERROR "libva not found: building HAVE_GAPI_ONEVPL without libVA support is impossible on UNIX systems")
-    endif(WITH_VA)
+  if(UNIX AND HAVE_VA)
+    ocv_target_include_directories(${the_module} SYSTEM PRIVATE ${VA_INCLUDE_DIR})
+    ocv_target_include_directories(opencv_test_gapi SYSTEM PRIVATE ${VA_INCLUDE_DIR})
```

Comment:
> opencv_test_gapi

Test target should be guarded, see code below.

Reply:
Good point, updated. Thanks